spec/features/admin_cant_delete_himself_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+feature 'AdminCanDeleteUser'  do
```

Comment:
It can be inside one spec ->
``` Feature manage_users_by_admin ````
1st scenario ``` admin can delete user ```
2nd scenario ``` admin can't delete himself ```